config/srcSchemaMap.py

```diff
@@ -0,0 +1,209 @@
+"base_PixelFlags_flag": "no_equivalent",
+"base_PixelFlags_flag_edge": "flag_pixel_edge",
+"base_PixelFlags_flag_offimage": "flag_pixel_offimage",
+"base_PixelFlags_flag_interpolated": "flag_pixel_interpolated_any",
+"base_PixelFlags_flag_interpolatedCenter": "flag_pixel_interpolated_center",
+"base_PixelFlags_flag_saturated": "flag_pixel_saturated_any",
+"base_PixelFlags_flag_saturatedCenter": "flag_pixel_saturated_center",
+"base_PixelFlags_flag_suspect": "flag_pixel_suspect_any",
+"base_PixelFlags_flag_suspectCenter": "flag_pixel_suspect_center",
+"base_PixelFlags_flag_cr": "flag_pixel_cr_any",
+"base_PixelFlags_flag_crCenter": "flag_pixel_cr_center",
+"base_PixelFlags_flag_bad": "flag_pixel_bad",
+"base_PsfFlux_flux": "flux_psf",
+"base_PsfFlux_fluxSigma": "flux_psf_err",
+"base_PsfFlux_flag": "flux_psf_flag",
+"base_PsfFlux_flag_noGoodPixels": "no_equivalent",
+"base_PsfFlux_flag_edge": "no_equivalent",
+"ext_photometryKron_KronFlux_flux": "flux_kron",
+"ext_photometryKron_KronFlux_fluxSigma": "flux_kron_err",
+"ext_photometryKron_KronFlux_radius": "flux_kron_radius",
+"ext_photometryKron_KronFlux_radius_for_radius": "flux_kron_radiusForRadius",
+"ext_photometryKron_KronFlux_psf_radius": "flux_kron_psfRadius",
+"ext_photometryKron_KronFlux_flag": "flux_kron_flag",
+"ext_photometryKron_KronFlux_flag_edge": "flux_kron_flag_edge",
+"ext_photometryKron_KronFlux_flag_bad_shape_no_psf": "no_equivalent",
+"ext_photometryKron_KronFlux_flag_no_minimum_radius": "no_equivalent",
+"ext_photometryKron_KronFlux_flag_no_fallback_radius": "no_equivalent",
+"ext_photometryKron_KronFlux_flag_radius": "flux_kron_flag_radius",
+"ext_photometryKron_KronFlux_flag_used_minimum_radius": "flux_kron_flag_usedMinimumRadius",
+"ext_photometryKron_KronFlux_flag_used_psf_radius": "flux_kron_flag_usedPsfRadius",
+"ext_photometryKron_KronFlux_flag_small_radius": "flux_kron_flag_smallRadius",
+"ext_photometryKron_KronFlux_flag_bad_shape": "flux_kron_flag_badShape",
+"base_FPPosition_x": "focalplane_x",
+"base_FPPosition_y": "focalplane_y",
+"base_FPPosition_flag": "no_equivalent",
+"base_Jacobian_value": "jacobian",
+"base_Jacobian_flag": "no_equivalent",
+"base_ClassificationExtendedness_value": "classification_extendedness",
+"base_ClassificationExtendedness_flag": "no_equivalent",
+"base_PsfFlux_apCorr": "flux_psf_apcorr",
+"base_PsfFlux_apCorrSigma": "flux_psf_apcorr_err",
+"base_PsfFlux_flag_apCorr": "flux_psf_flag_apcorr",
+"ext_photometryKron_KronFlux_apCorr": "flux_kron_apcorr",
+"ext_photometryKron_KronFlux_apCorrSigma": "flux_kron_apcorr_err",
+"ext_photometryKron_KronFlux_flag_apCorr": "flux_kron_flag_apcorr",
+"base_GaussianFlux_apCorr": "flux_gaussian_apcorr",
+"base_GaussianFlux_apCorrSigma": "flux_gaussian_apcorr_err",
+"base_GaussianFlux_flag_apCorr": "flux_gaussian_flag_apcorr",
+"calib_psfUsed": "calib_psf_used",
+"calib_psfCandidate": "calib_psf_candidate",
+"calib_psfReserved": "calib_psf_reserved",
+}
+
+#========== No equivalents below =======
```

Comment:
Correct...and unnecessary!  I've removed the commented out lines and added a few explanatory notes at the top of the file.